retrofit-converters/logansquare/src/test/java/retrofit/converter/LoganSquareConverterTest.java

```diff
@@ -0,0 +1,67 @@
+package retrofit.converter;
+
+
+import com.bluelinelabs.logansquare.NoSuchMapperException;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonParser;
+import com.squareup.okhttp.MediaType;
+import com.squareup.okhttp.RequestBody;
+import com.squareup.okhttp.ResponseBody;
+
+import org.junit.Test;
+
+import java.io.IOException;
+
+import okio.Buffer;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LoganSquareConverterTest {
+
+  private static final MediaType MEDIA_TYPE = MediaType.parse("application/json; charset=UTF-8");
+  private static final MyObject OBJECT = new MyObject("hello world", 10);
+  private final String JSON = "{\"message\":\"hello world\",\"count\":10}";
+  private LoganSquareConverter converter = new LoganSquareConverter();
+
+  @Test
+  public void serialize() throws Exception {
+    RequestBody body = converter.toBody(OBJECT, MyObject.class);
+    assertThat(body.contentType()).isEqualTo(MEDIA_TYPE);
+    assertForEquality(body, JSON);
+  }
+
+  @Test
+  public void deserialize() throws Exception {
+    ResponseBody body = ResponseBody.create(MEDIA_TYPE, JSON);
+    MyObject result = (MyObject) converter.fromBody(body, MyObject.class);
+    assertThat(result).isEqualTo(OBJECT);
+  }
+
+  @Test
+  public void deserializeWrongValue() throws Exception {
+    ResponseBody body = ResponseBody.create(MEDIA_TYPE, "{\"foo\":\"bar\"}");
+    try {
+      converter.fromBody(body, MyObject.class);
+    } catch (Exception ignored) {
```

Comment:
@JakeWharton What's your view about the [discussion](https://github.com/bluelinelabs/LoganSquare/issues/64#issuecomment-112162065) regarding adding `fail()` at the end of the try block?